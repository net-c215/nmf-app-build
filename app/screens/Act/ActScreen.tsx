import React from "react";
import { ScrollView, StyleSheet } from "react-native";

import { t } from "../../utils/translations";

const styles = StyleSheet.create({
  container: {
    flex: 1,
    paddingTop: 15,
    backgroundColor: "#fff"
  }
});

export default function ActScreen(): React.ReactElement {
  return <ScrollView style={styles.container}></ScrollView>;
}

ActScreen.navigationOptions = {
  title: t("ACT")
};
