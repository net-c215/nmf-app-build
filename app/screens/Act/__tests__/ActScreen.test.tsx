import React from "react";
import renderer from "react-test-renderer";
import ActScreen from "../ActScreen";

const props = {
  navigation: {
    push: () => {
      // do nothing.
    }
  }
};

it("EmissionsScreen renders correctly", () => {
  const tree = renderer.create(<ActScreen {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
