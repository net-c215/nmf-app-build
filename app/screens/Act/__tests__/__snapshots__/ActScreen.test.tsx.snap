// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActScreen renders correctly 1`] = `
<RCTScrollView
  data={
    Array [
      Object {
        "body": "<h3>Section</h3>
<p>Hello world, some markdown here</p>
<p><img src=\\"restaurant.png\\" alt=\\"Restaurant\\"></p>
<p><img src=\\"bike.png\\" alt=\\"Bike\\"></p>
",
        "category": "technology",
        "key": ".//find-replacement-for-GAFA-services.md",
        "title": "Find a replacement for GAFA services",
      },
      Object {
        "body": "<h3>Section</h3>
<p>Hello world, some markdown here</p>
<p><img src=\\"bike.png\\" alt=\\"Bike\\"></p>
<p>Yeah, NFM on Bike!</p>
",
        "category": "technology",
        "key": ".//clean-your-mailbox.md",
        "title": "Clean your mailbox",
      },
      Object {
        "body": "<h3>Section</h3>
<p>Hello world, some markdown here</p>
<p><img src=\\"bike.png\\" alt=\\"Bike\\">
<img src=\\"earth.png\\" alt=\\"Earth\\">
<img src=\\"restaurant.png\\" alt=\\"Restaurant\\"></p>
",
        "category": "technology",
        "key": ".//install-adblocker.md",
        "title": "Install an Adblocker",
      },
    ]
  }
  disableVirtualization={false}
  getItem={[Function]}
  getItemCount={[Function]}
  horizontal={false}
  initialNumToRender={10}
  keyExtractor={[Function]}
  maxToRenderPerBatch={10}
  onContentSizeChange={[Function]}
  onEndReachedThreshold={2}
  onLayout={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={Array []}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "paddingHorizontal": 14,
    }
  }
  updateCellsBatchingPeriod={50}
  viewabilityConfigCallbackPairs={Array []}
  windowSize={21}
>
  <View>
    <View
      onLayout={[Function]}
      style={null}
    >
      <TouchableOpacity
        onPress={[Function]}
        style={
          Object {
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <Text.Secondary
          numberOfLines={1}
          style={
            Object {
              "paddingVertical": 20,
            }
          }
        >
          Find a replacement for GAFA services
        </Text.Secondary>
        <Ionicons
          color="#9CA6A0"
          name="ios-arrow-forward"
          size={26}
          style={
            Object {
              "marginTop": 18,
            }
          }
        />
      </TouchableOpacity>
    </View>
    <View
      onLayout={[Function]}
      style={null}
    >
      <TouchableOpacity
        onPress={[Function]}
        style={
          Object {
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <Text.Secondary
          numberOfLines={1}
          style={
            Object {
              "paddingVertical": 20,
            }
          }
        >
          Clean your mailbox
        </Text.Secondary>
        <Ionicons
          color="#9CA6A0"
          name="ios-arrow-forward"
          size={26}
          style={
            Object {
              "marginTop": 18,
            }
          }
        />
      </TouchableOpacity>
    </View>
    <View
      onLayout={[Function]}
      style={null}
    >
      <TouchableOpacity
        onPress={[Function]}
        style={
          Object {
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <Text.Secondary
          numberOfLines={1}
          style={
            Object {
              "paddingVertical": 20,
            }
          }
        >
          Install an Adblocker
        </Text.Secondary>
        <Ionicons
          color="#9CA6A0"
          name="ios-arrow-forward"
          size={26}
          style={
            Object {
              "marginTop": 18,
            }
          }
        />
      </TouchableOpacity>
    </View>
  </View>
</RCTScrollView>
`;
